CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,59 @@
+- [Italian/Italiano](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/it.md)
+- [Japanese/日本語](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/jp.md)
+- [Korean/한국어](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/ko.md)
+- [Ukrainian/Українська](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/uk.md)
+- [Russian/Русский](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/ru.md)
+- [Portuguese/Português](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/pt.md)
+- [Arabic/العربية](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/ar.md)
+- [Polish/Polski](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/pl.md)
+- [Indonesian/Bahasa Indonesia](https://raw.githubusercontent.com/cncf/foundation/master/code-of-conduct-languages/id.md)
+
+### Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of fostering
+an open and welcoming community, we pledge to respect all people who contribute
+through reporting issues, posting feature requests, updating documentation,
+submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for
+everyone, regardless of level of experience, gender, gender identity and expression,
+sexual orientation, disability, personal appearance, body size, race, ethnicity, age,
+religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing others' private information, such as physical or electronic addresses,
+ without explicit permission
+* Other unethical or unprofessional conduct.
+
+Project maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are not
+aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers
+commit themselves to fairly and consistently applying these principles to every aspect
+of managing this project. Project maintainers who do not follow or enforce the Code of
+Conduct may be permanently removed from the project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior in Kubernetes may be reported by contacting the [Kubernetes Code of Conduct Committee](https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. For other projects, please contact a CNCF project maintainer or our mediator, Mishi Choudhary <mishi@linux.com>.
```

Comment:
CNCF code of conduct issues should be reported to k8s code of conduct? :thinking:

Reply:
Sigh, yes. Yet they use Slack which has no block button. It's so disappointing.

Reply:
hmm OK :) just checking :+1: